src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: A "NullPointerException" could be thrown; "message" is nullable here. ([squid:S2259](http://localhost:9000/coding_rules#rule_key=squid%3AS2259))

[See it in SonarQube](http://localhost:9000/project/issues?id=org.sonarqube%3Apr-decoration&branch=pr_29&issues=AWBUGpoGvB3VRKxIB7gR&open=AWBUGpoGvB3VRKxIB7gR)

[ISSUE_KEY]: AWBUGpoGvB3VRKxIB7gR

Reply:
===RESOLVED===
This issue has been resolved by a user in SonarQube